step-capstone/src/scripts/Categories.js

```diff
@@ -0,0 +1,80 @@
+const shopping = new Set(["bicycle_store",
+    "book_store",
+    "clothing_store",
+    "jewelry_store",
+    "florist",
+    "shopping_mall",
+    "shoe_store",
+    "store"
+])
+const beauty = new Set([
+    "beauty_salon",
+    "hair_care",
+    "spa"])
+
+const entertainment = new Set(["Bowling_alley",
```

Comment:
I see a bunch of capitalization inconsistencies within the file "Bowling_alley" instead of "bowling_alley", "Movie_theater" instead of "movie_theater". Please fix them

Reply:
thank you, we've addressed that

Reply:
fixed